packages/flutter/lib/src/painting/debug.dart

```diff
@@ -36,6 +36,12 @@ typedef HttpClientProvider = HttpClient Function();
 /// This value is ignored in non-debug builds.
 HttpClientProvider? debugNetworkImageHttpClientProvider;
 
+/// If a background isolate was used to decode an image, this will contain
```

Comment:
nit: fetch (and consolidate)?

We're not actually decoding the image on the background isolate, although decoding always happens on some background thread.